src/device-manager/java/src/nl/Weave/DataManagement/WDMClient.java

```diff
@@ -0,0 +1,37 @@
+/*
+
+    Copyright (c) 2019 Google, LLC.
+    All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+package nl.Weave.DataManagement;
+
+public interface WDMClient
+{
+    public void connect(long deviceMgrPtr);
```

Comment:
Can we remove this from the interface? Since this is only required the `WeaveDeviceManager` based `WDMClientImpl`?

Reply:
Done, thanks